.github/workflows/release-image.yml

```
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2025 Canonical Ltd.
+name: Release Docker Image
+
+on:
+  workflow_run:
+    workflows: ["Tag GitHub"]
```

What about something like this?
```suggestion
    workflows:
        - "Tag GitHub"
```

---

Done